docs/source/overview/delivery_services.rst

```diff
@@ -697,6 +697,12 @@ Servers
 -------
 Servers can be assigned to Delivery Services using the :ref:`tp-configure-servers` and :ref:`tp-services-delivery-service` Traffic Portal sections, or by directly using the :ref:`to-api-deliveryserviceserver` endpoint. Only :term:`Edge-tier cache servers` can be assigned to a Delivery Service, and once they are so assigned they will begin to serve content for the Delivery Service (after updates are queued and then applied). Any servers assigned to a Delivery Service must also belong to the same CDN_ as the Delivery Service itself. At least one server must be assigned to a Delivery Service in order for it to serve any content.
 
+.. _ds-service-category:
+
+Service Category
+----------------
+Defines the type of content being delivered by the Delivery Service. Some example values are: "Linear", "VOD", and "OTT"
```

Comment:
Paragraphs should consist of complete sentences (missing punctuation)

Also, I think this explanation could be clearer. This doesn't "define" the type of content because it has no semantic value. This is essentially just like a "tag" for the Delivery Service that more qualitatively "describes" the type of its content.

(also also: pls drop that last example as it's very Comcast-specific and nobody should need to know or even wonder what that could be)